setup.py

```diff
@@ -25,24 +28,7 @@
     ],
     zip_safe             = False,
     python_requires      = '>=3.7.4',
-    install_requires     = [
-        "speedrun @ git+https://github.com/inferno-pytorch/speedrun.git@p2p_risk_prediction#egg=speedrun",
-        "ctt @ git+https://github.com/nasimrahaman/ctt@master#egg=ctt",
```

Comment:
the "#egg=<pkgname>" suffix might be missing from the reqs.txt file; I'm not sure what the difference is, but I've only seen examples online with the egg thing (and the <pkgname @> prefix) if it's used here

Reply:
I've used the content of setup.py in requirements.txt but I've just added a comment in the .txt to explain the format it should follow with git end points